submodules/k8s/main.tf

```diff
@@ -23,8 +23,8 @@ locals {
         aws_auth_yaml       = basename(local.aws_auth_filename)
         eniconfig_yaml      = local.eniconfig_filename != "" ? basename(local.eniconfig_filename) : ""
         calico_operator_url = local.calico.operator_url
-        bastion_user        = var.bastion_user
-        bastion_public_ip   = var.bastion_public_ip
+        bastion_user        = var.bastion_user != null ? var.bastion_user : ""
```

Comment:
`coalesce(var.bastion_user, "")`?

Reply:
coalesce ignores both `null` and `""` 
```
> coalesce(null, "")
╷
│ Error: Error in function call
│
│   on <console-input> line 1:
│   (source code not available)
│
│ Call to function "coalesce" failed: no non-null, non-empty-string arguments.
╵
```
we need the empty string to make it to the template

Reply:
yep, my bad